vsgenstubs/_vapoursynth.part.pyi

```diff
@@ -270,23 +268,24 @@ Func = typing.Callable[..., typing.Any]
 
 class Error(Exception): ...
 
-def set_message_handler(handler_func: typing.Callable[[int, str], None]) -> None: ...
-def clear_output(index: int = 0) -> None: ...
-def clear_outputs() -> None: ...
+def set_message_handler(handler_func: typing.Callable[[int, str], None]) -> typing.NoReturn: ...
```

Comment:
According to the typing-documentation, NoReturn denotes a function that never returns (throws, terminates the process/thread, etc...)

Reply:
I dropped the commit as I misread the usecase for typing.NoReturn as a function that just doesn't have a return statement, not one that *never* returns.